storm-server/src/main/java/org/apache/storm/daemon/supervisor/BasicContainer.java

```diff
@@ -346,7 +346,7 @@ protected String getWildcardDir(File dir) {
     }
     
     protected List<String> frameworkClasspath(SimpleVersion topoVersion) {
-        File stormWorkerLibDir = new File(_stormHome, "lib-worker");
+        File stormWorkerLibDir = new File(_stormHome, "lib");
```

Comment:
Will revert it. After speaking to @HeartSaVioR figured out the build issue on my end that was needing this change. Its not needed